From hwy@cn.fujitsu.com  Wed Oct  1 13:58:58 2008
From: "Huang Weiyi" <hwy@cn.fujitsu.com>
Date: Thu, 25 Sep 2008 13:11:28 +0800
Subject: USB: ti_usb_3410_5052: removed duplicated include
To: <gregkh@suse.de>
Message-ID: <000701c91ecd$2a8184f0$248da70a@hwysdesktop>


Removed duplicated #include <linux/firmware.h> in
drivers/usb/serial/ti_usb_3410_5052.c.

Signed-off-by: Huang Weiyi <hwy@cn.fujitsu.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/ti_usb_3410_5052.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/serial/ti_usb_3410_5052.c
+++ b/drivers/usb/serial/ti_usb_3410_5052.c
@@ -85,7 +85,6 @@
 #include <linux/uaccess.h>
 #include <linux/usb.h>
 #include <linux/usb/serial.h>
-#include <linux/firmware.h>
 
 #include "ti_usb_3410_5052.h"
 
